platform/android/export/export_plugin.cpp

```diff
@@ -1698,6 +1698,91 @@ void EditorExportPlatformAndroid::get_preset_features(const Ref<EditorExportPres
 	}
 }
 
+String EditorExportPlatformAndroid::get_export_option_warning(const EditorExportPreset *p_preset, const StringName &p_name) const {
```

Comment:
The checks here replicate the ones in `has_valid_project_configuration(...)`. Can the latter method be deprecated then?

Reply:
I see from your example screenshots they serve different purposes. It may be easier then to move the warnings check logic into a helper method that both methods use.

Reply:
Yes, probably it can be merged. I guess I'll add an extra `validation_required` option (to indicate what's warning and what's error) to the `ExportOption`, and check all the options using `get_export_option_warning` instead of duplicating the checks in the `has_valid_project_configuration`.

Reply:
Updated to use `get_export_option_warning` for both.